pkg/resources/view_acceptance_test.go

```diff
@@ -16,31 +24,42 @@ import (
 	"github.com/hashicorp/terraform-plugin-testing/tfversion"
 )
 
-func TestAcc_View(t *testing.T) {
-	viewId := acc.TestClient().Ids.RandomSchemaObjectIdentifier()
-	accName := viewId.Name()
-	query := "SELECT ROLE_NAME, ROLE_OWNER FROM INFORMATION_SCHEMA.APPLICABLE_ROLES"
-	otherQuery := "SELECT ROLE_NAME, ROLE_OWNER FROM INFORMATION_SCHEMA.APPLICABLE_ROLES where ROLE_OWNER like 'foo%%'"
+func TestAcc_View_basic(t *testing.T) {
+	_ = testenvs.GetOrSkipTest(t, testenvs.EnableAcceptance)
+	acc.TestAccPreCheck(t)
 
-	m := func() map[string]config.Variable {
-		return map[string]config.Variable{
-			"name":        config.StringVariable(accName),
-			"database":    config.StringVariable(acc.TestDatabaseName),
-			"schema":      config.StringVariable(acc.TestSchemaName),
-			"comment":     config.StringVariable("Terraform test resource"),
-			"is_secure":   config.BoolVariable(true),
-			"or_replace":  config.BoolVariable(false),
-			"copy_grants": config.BoolVariable(false),
-			"statement":   config.StringVariable(query),
+	rowAccessPolicy, rowAccessPolicyCleanup := acc.TestClient().RowAccessPolicy.CreateRowAccessPolicyWithDataType(t, sdk.DataTypeVARCHAR)
+	t.Cleanup(rowAccessPolicyCleanup)
+
+	aggregationPolicy, aggregationPolicyCleanup := acc.TestClient().AggregationPolicy.CreateAggregationPolicy(t)
+	t.Cleanup(aggregationPolicyCleanup)
+
+	rowAccessPolicy2, rowAccessPolicy2Cleanup := acc.TestClient().RowAccessPolicy.CreateRowAccessPolicyWithDataType(t, sdk.DataTypeVARCHAR)
+	t.Cleanup(rowAccessPolicy2Cleanup)
+
+	aggregationPolicy2, aggregationPolicy2Cleanup := acc.TestClient().AggregationPolicy.CreateAggregationPolicy(t)
+	t.Cleanup(aggregationPolicy2Cleanup)
+
+	id := acc.TestClient().Ids.RandomSchemaObjectIdentifier()
+	statement := "SELECT ROLE_NAME, ROLE_OWNER FROM INFORMATION_SCHEMA.APPLICABLE_ROLES"
+	otherStatement := "SELECT ROLE_NAME, ROLE_OWNER FROM INFORMATION_SCHEMA.APPLICABLE_ROLES where ROLE_OWNER like 'foo%%'"
+
+	viewModel := model.View("test", id.DatabaseName(), id.Name(), id.SchemaName(), statement)
+
+	// generators currently don't handle lists, so use the old way
```

Comment:
Is there a ticket for generator improvements, if yes let's put it here

Reply:
But you can add your own non-generated implementation for the list fields in _ext.go file. There is a TODO for supporting generated lists already but it should be possible to add the quickly as ext now, so please try doing that.

Reply:
This limitation is already noted in this generator. I think that with `config.Variable` you can't pass nested objects, so ViewModel needs adjustments.

Reply:
Why not? What about `MapVariable` and `ListVariable`? The purpose of using config.Variable interface was to be able to specify anything we were able to specify using `config.Variables`

Reply:
We discussed verbally, we postpone this due to current behavior of list of objects (json).